### DISCUSSION TITLE
Example with Honojs - A beautiful combination of modern web

### DISCUSSION THREAD

**Discussion by nhattran998** [vikejs/vike#1381] [Discussion]
Hi Vike Team, I see your team already discuss with Honojs Team in the pass, but I cannot see any example of the combination: Vike + Honojs.

I [reproduce a simple project](https://github.com/harrytran998/repro-hono-vike/blob/main/server/index.ts), but seem no work 😆 .
Do you have any suggestion?

<img width="986" alt="image" src="https://github.com/vikejs/vike/assets/24296018/61f5851e-d4f5-481f-ad7f-3e5b3de030ad">

Thank team <3

**Comment by brillout** [comment]
Hi Harry. I think the question is how to make Vite work with Hono, as Vike itself shouldn't cause any issues. And I believe Hono has some Vite integration packages?

I think if we get a simple Hono + Vite (with Vite in middleware mode, and without Vike) example going, then adding Vike is straighforward, right?

**Reply by nhattran998** [reply]
Yeah - currently, Hono focus on support Cloudflare or Edge something --> They already had the Vite Plugin, but currently not work well enough to serve all kinds of Vite projects. So might be we need to defer this idea - back to basic server like h3, express, fastify... 😆.

**Comment by brillout** [comment]
Let's consolidate the discussion over here: https://github.com/vikejs/vike/discussions/1391.
